validator/README.md

```diff
@@ -66,22 +66,26 @@ try to port to earlier versions of NodeJS if sufficient interest exists.
 
 ```js
 'use strict';
-
 const ampValidator = require('amp-validator');
-ampValidator.getInstance((instance) => {
-  const result = instance.validateString('<html>Hello, world</html>');
-  ((result.status === 'PASS') ? console.log : console.error)(result.status);
-  for (const error of result.errors) {
-    let msg = 'line ' + error.line + ', col ' + error.col + ': ' +
-        error.message;
-    if (error.specUrl !== null) {
-      msg += ' (see ' + error.specUrl + ')';
-    }
-    ((error.severity === 'ERROR') ? console.error : console.warn)(msg);
-  }
+
+var dataFails = '<html amp>Hello,world</html';
+var ourInstance = ampValidator.getInstance()
```

Comment:
ourInstance isn't necessary here, please get rid of the extra variable. It can just be

`ampValidator.getInstance().then(...`